client/src/components/AlumniProfile.js

```diff
@@ -286,28 +246,149 @@ export default class AlumniProfile extends Component {
                     size="medium"
                     src={details.imageURL}
                 />
+                {
+                !isViewOnly ?
+                    <>
+                        <Button
+                        style={{'width': '30%', 'align-self': 'center', 'margin': '5px 0 5px 0'}}
+                        basic
+                        color="blue"
+                        onClick={this.openImageModal}
+                        >
+                            Update Image
+                        </Button>
+                        <ImageUpdateModal
+                            modalOpen={this.state.imageModalOpen}
+                            id={details._id}
+                            isAlumni={true}
+                            closeModal={this.closeImageModal}
+                        />
+                    </>
+                    : null
+                }
                 <Card.Content>
                     <Card.Header>{details.name || 'Unavailable'}</Card.Header>
-                    <Card.Meta>{details.jobTitleName || 'Unavailable'}</Card.Meta>
-
-                    <Card.Description>College: {details.collegeName || 'Unavailable'}</Card.Description>
-                    <Card.Description>Location: {(details.city && details.country) ? `${details.city} (${details.country})` : 'Unavailable'}</Card.Description>
-                    <Card.Description>Company: {details.companyName || 'Unavailable'}</Card.Description>
-                    <Card.Description>Major: {details.majorName || 'Unavailable'}</Card.Description>
+                    <Card.Meta>{details.jobTitleName || 'Unavailable'}
+                        {
+                            !isViewOnly ?
+                                <>
+                                    <Button
+                                        style={{'margin': '0 0 2px 2px'}}
+                                        icon
+                                        basic
+                                        color="blue"
+                                        size="mini"
+                                        onClick={this.openJobTitleModal}
+                                    >
+                                        <Icon name='pencil'/>
+                                    </Button>
```

Comment:
Design comment - to remove borders, use
`  
<Button icon inverted>
    <Icon name='pencil' color='blue' />
</Button>
`

![image](https://user-images.githubusercontent.com/54961598/85930063-67cff880-b887-11ea-81d0-6711929680dd.png)

Will remove clutter caused by the basic borders

Reply:
good call, looks cleaner with the changes pushed